cob_hardware_config/cob4-3/config/base_driver.yaml

```diff
@@ -0,0 +1,54 @@
+defaults:
+  eds_pkg: cob_hardware_config
+  eds_file: "cob4-3/config/Elmo.dcf"
+  dcf_overlay:
+    "6083": "1000000"
+    "6084": "1000000"
+    "60C5": "1000000"
+    "60C6": "1000000"
+    "6099sub1": "100000" # find switch (coarse)
+    "6099sub2": "10000" # find home (fine)
+    "1016sub1" : "0x7F0064" # heartbeat timeout of 100 ms for master at 127
+
+drive_wheel: &drive_wheel
+  dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+    "6098": "35" # homing method
+  vel_from_device: "p1 == nan || obj2041 == t1 ? 0 : deg2rad(1000*norm(obj6064-p1,-36000000,36000000)/norm(obj2041-t1,0,2^32)), p1=p0, p0 = obj6064, t1 = t0, t0 = obj2041"
```

Comment:
@ipa-mdl 
`p1 == nan` should be replaced by `p1 != p1` as we did it for the cob4-2 torso?

`vel_from_device: "v != v ? v=0 : v=smooth(deg2rad(obj606c/1000.0),v,0.3)" # for smoother current velocity in joint_states`